### ISSUE TITLE
Slave bot stuck at waiting for attack button to be active

### ISSUE THREAD

**Issue by greatmany** [Masuzu/ZooeyBot#57]
Slave mode calibrates 6/6 properly, bot will refill AP, clear network, start the battle, then run the script for the slave, then when it goes to main window it is stuck at waiting for attack button to be active (window is already completely loaded and ready).  After a while it'll timeout and start going into a refresh loop for both windows.

Bot runs fine in event mode so i am not sure what might be causing this.

Main account window is in focus when it is waiting for attack button to be active. If I manually use the keyboard shortcuts to attack using the main account the bot will resume normal operation again once it refreshes the windows, and go back to the room, start a new quest again, ready for both windows, and everything is fine until it is in battle, then it gets stuck at main account again.

To summarize what happens:
1) calibration ok, autostart battle
2) slave runs script to attack
3) main window is now active, but stuck at waiting for attack button
4) refresh loop

**Comment by greatmany** [comment]
After further testing I actually got it to work now by moving the two windows much further from each other. Basically I put them on the extreme left and extreme right ends of a 2560x1440 screen and it detects the attack button now. When I tried to align them similar to the example picture it would not work.

**Comment by greatmany** [comment]
I've tried it again under the same circumstances where I got it to work the last time but it is stuck waiting for attack button to be active again, I guess the position of the window was not a factor

**Comment by greatmany** [comment]
![image](https://cloud.githubusercontent.com/assets/28790334/26276187/dc1bbcee-3da4-11e7-94cd-d7712ef5c16f.png)
screenshot of console in case you need it

**Comment by Masuzu** [comment]
Hello,

After investigating the problem, the root of the issue would be that the main window does not seem to have the focus (alt-enter switches the "focus" on different windows), resulting in a loop causing the attack button not being detected on the main window while it is displayed.
I will make a patch for this as soon as possible.

**Comment by Masuzu** [comment]
This should now be addressed with version 1.6.2:
https://poker-bot.azurewebsites.net/ZooeyBot/en/Home/PatchNotes
